index.js

```diff
@@ -1,17 +1,55 @@
 export const getUserChoice = (userInput) => {
-  // TODO
+  // bo'sh joylarni o'chiramiz va kichik harfga o'tkazamiz.
+  let input = userInput.trim().toLowerCase();
+  // shartlarni birin-ketinlik bilan tekshiramiz
+  if (input === 'rock') {
+    return input;
+  } else if(input === 'paper') {
+    return input;
+  } else if(input === 'scissors') {
+    return input;
+  } else if(input === 'bomb') {
+    return input;
+  }
+  // shartlarni birortasi qanoatlantirmasa, console da Error_message yozuvi paydo bo'ladi
+   else {
+    console.log('Error_message');
+  }
 };
 
 export const getComputerChoice = () => {
-  // TODO
+  // 0 va 3 sonlar orasidan tasodifiy butun sonni qaytaramiz
+  let randomIndex = Math.floor(Math.random() * 3);
+  // Massiv elementlarini kiritib olamiz
+  let element = ['rock','paper','scissors'];
```

Comment:
`element` bu birlikdagi bitta element degani. Ko'plikda `elements` ("elementlar") deyish to'g'ri bo'lardi. Yanayam to'g'riroq nomlar yo `choices` yoki `options` bo'lishi to'g'riroq fikrimcha.